helm/up9/templates/up9-clusterrole.yaml

```diff
@@ -15,3 +15,27 @@ rules:
   - apiGroups: [ "", "extensions" ]
     resources: [ "configmaps" ]
     verbs: [ "patch", "get", "update" ]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: {{ .Values.clusterRoleBindingName }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: {{ .Values.clusterRoleName }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.serviceAccountName }}
+    namespace: {{ .Values.namespace  }}
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ .Values.serviceAccountName }}
+  namespace: {{ .Values.namespace }}
+  labels:
+    kubernetes.io/cluster-service: "true"
+    addonmanager.kubernetes.io/mode: Reconcile
+automountServiceAccountToken: true
```

Comment:
What does this do?

Reply:
it takes the value from here if the resource does not override this.
and we do want to automount the service account token when we are using this service account. so I removed it from the deployment and daemon set because it can be here